Q_AND_A.md

```diff
@@ -79,3 +79,6 @@
   - `write` - allows a DID to create and update the record they have created
   - `update` - allows a DID to update a record, regardless of the initial author
 
+  ## Subscriptions
+ - Same considerations as for `RecordsQuery` wrt invoking a protocol role for a specific `protocolPath`.
+ - Currently if a subscription is no longer authorized but it is still active, the subscriber will still receive updates until they close the subscription. This will be addressed ina separate PR https://github.com/TBD54566975/dwn-sdk-js/issues/668 - last updated (2024/01/22)
```

Comment:
A few thoughts:
1. Entries ought to follow the same "question followed by an answer" here, the intent of this document was mainly to capture "why questions" that people/we ask.
1. If this is more about documenting design, [technical-proposals](https://github.com/TBD54566975/technical-proposals) seems to be a much better place. I also understand if a total separate technical proposal seems like an overkill at times, in which case, consider a separate light weight `design.md` that captures all feature designs that fall in this category. I'd be down if you'd like to combine both Q&A and design into one document, in which case, let's rename to `design.md` and have two sections in the document: "Design" and "Q&A" sections.

Reply:
Makes sense! Maybe I will re-word it to question and answer for this case?

The intent was to document this behavior somewhere until we implement it as I think it's something that people would wonder about when using this. I'm pretty sure it will not be long before it's implemented, or it be troublesome, but wanted to make sure we get certain functionality in place before going down this route.

Maybe some wording like:
```suggestion
  ## Subscriptions
 - What happens to a subscription which is listening to events, but is no longer authorized due to revocation of a grant or role? 
 
Currently if a subscription is no longer authorized but it is still active, the subscriber will still receive updates until they close the subscription.
This will be addressed in a future upgrade and we've created an issue to track it. https://github.com/TBD54566975/dwn-sdk-js/issues/668 - last updated (2024/01/22)
```

Reply:
Got it, given the intent and the way you have reworded it, the Q&A is a great place to include it.

Reply:
Fixed! Added `RecordsSubscribe` to the protocol answer that relates to the `protocolPath`. Let me know if that reads better and makes sense.